### PR TITLE
[v1 API Docs] add */property/values pages

### PR DESCRIPTION
**Pull request by juliawu**
Adds the following v1 REST API documentation pages:
* v1/property/values
* v1/bulk/property/values

### CODE REVIEW

File: api/rest/v1/bulk_property_values.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+nav_exclude: true
+parent: v1 REST
+grand_parent: API
+published: false
+permalink: /api/rest/v1/bulk/property/values
+---
+
+# /v1/bulk/property/values
+
+Get the values of a [property](/glossary.html#property) for multiple entities.
```
Comment (shifucun) [inline api/rest/v1/bulk_property_values.md:13]: please change "entities" to "nodes" throughout this change

File: api/rest/v1/bulk_property_values.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+Data Commons represents properties as labels of directed edges between nodes,
+where the successor node is a value of the property. Thus, this endpoint returns
```
Comment (shifucun) [inline api/rest/v1/bulk_property_values.md:16]: nit: lint on long lines

**Review by shifucun — APPROVED** [review]
A major todo would be adding pagination explanation /example for this API. Could be done in a follow up PR

File: api/rest/v1/bulk_property_values.md (hunk 2 above)
Comment (juliawu) [reply api/rest/v1/bulk_property_values.md:16]: linter already applied at line-length=80

File: api/rest/v1/bulk_property_values.md (hunk 1 above)
Comment (juliawu) [reply api/rest/v1/bulk_property_values.md]: Done!

**Pull request merged**